content/en/apps/reference/app-settings/transitions.md

```diff
@@ -34,7 +34,7 @@ The following transitions are available and executed in order.
 | Key | Description |
 |---|---|
 | maintain_info_document | Records metadata about the document such as when it was replicated. Enabled by default. |
-| update_clinics | Adds a contact's info to a new data record. This is used to attribute an incoming SMS message or report to the appropriate contact. The `rc_code` value on the contact is used to match to the value of the form field set as the `facility_reference` in the [JSON form definition]({{< ref "apps/reference/app-settings/forms#app_settingsjson-forms" >}}). This matching is useful when reports are sent on behalf of a facility by unknown or various phone numbers. If `facility_reference` is not set for a form, the contact match is attempted using the sender's phone number. |
+| update_clinics | Adds a contact's info to a new data record. This is used to attribute an incoming SMS message or report to the appropriate contact. The `rc_code` value on the contact is used to match to the value of the form field set as the `facility_reference` in the [JSON form definition]({{< ref "apps/reference/app-settings/forms#app_settingsjson-forms" >}}). This matching is useful when reports are sent on behalf of a facility by unknown or various phone numbers. If `facility_reference` is not set for a form, the contact match is attempted using the sender's phone number. If a form is not public and a match is not found then the `sys.facility_not_found` error is raised. Ammessage can be configured to be sent out whenever this error is raised. |
```

Comment:
I think we should add more details about how the configuration should look like, under which key it should be found, what happens if no configuration is provided and so on. Please have a look at the other transitions that have configuration and detailed descriptions, for example: https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#accept-patient-reports